tincr/cad/design/lib_cells.tcl

```diff
@@ -154,7 +155,7 @@ proc ::tincr::get_supported_libcells { } {
 #   as supported if, when instantiated, the reference name of the cell instance matches the name
```

Comment:
Seems like this is static information that doesn't change between successive invocations of this proc. That makes it a good candidate for the cache. I would create a cache called `list.supported.leaf.lib_cells` and add the logic in this proc to its `generate` script. Then you can add `save` and `load` scripts to persist this data to disk, eliminating the need to regenerate it on each invocation of this proc.